README.md

```diff
@@ -156,17 +156,13 @@ def test_that_only_runs_on_eos_on_7130(dut):
     logging.info("Must be EOS on 7130!")
 ```
 
-OS decorators accept the following keywords per OS:
+OS decorators accept the following keywords:
  - min_version
  - min_change_number
 
-Expected DUT verison format: ```<D.D.D>XX-XX-XX-<change_number>```
-I.e. series of dot separated decimals (min_version) and
-the last word after the last dash will be interpreted as the change number.
-
 If both kwargs are specified, min_change_number takes precedence.
 ```python
-@pytest.mark.eos(min_version="4.30.0")
+@pytest.mark.eos(min_version="4.30.0", min_change_number="3452345")
```

Comment:
Is there a reason we would like the change number to be a string? I think this could just be an int, which would be nicer.